calico_versioned_docs/version-3.31/_includes/partials/_system-requirements.mdx

```diff
@@ -9,15 +9,39 @@ import Link from '@docusaurus/Link';
   When IPIP is enabled (the default), Calico also needs to be able to manage `tunl*` interfaces.
   When VXLAN is enabled, Calico also needs to be able to manage the `vxlan.calico` interface.
 
-* Linux kernel 3.10 or later with [required dependencies](#kernel-dependencies).
-  The following distributions have the required kernel, its dependencies, and are known to work well with $[prodname].
+* A Linux distribution that has the minimum required Linux kernel version and [required dependencies](#kernel-dependencies).
+  If you use the eBPF data plane, a later version of the kernel is required.
+  * Minimum Linux kernel version: 5.10.
+  * Minimum Linux kernel version for eBPF: 5.17.
+
+  The following distributions have the required kernels and dependencies, and are known to work well with $[prodname].
   <If condition={props.platform === 'kubernetes'}>
-  * RedHat Linux 7
-  * CentOS 7
-  * Flatcar Container Linux
-  * Fedora CoreOS
-  * Ubuntu 18.04
-  * Debian 8
+  <table>
+    <thead>
+    <tr>
+      <th>Distribution</th>
+      <th>Compatible with Calico</th>
+      <th>Compatible with Calico eBPF</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td>RHEL</td>
+      <td>8 or later</td>
+      <td>9 or later</td>
+    </tr>
+    <tr>
+      <td>Ubuntu</td>
+      <td>22.04 LTS or later</td>
+      <td>24.04 LTS or later</td>
```

Comment:
We do run CI on 22.04 for ebpf. Ubuntu 22.04 on gcp comes with 6.8 kernel IIRC.  I think it is better to recommend to use later kernel versions rather than enforcing a strict minimum requirement. WDYT?